system/lib/standalone/__main_void.c

```diff
@@ -0,0 +1,52 @@
+/*
+ * Copyright 2019 The Emscripten Authors.  All rights reserved.
+ * Emscripten is available under two separate licenses, the MIT license and the
+ * University of Illinois/NCSA Open Source License.  Both these licenses can be
+ * found in the LICENSE file.
+ */
+
+#include <alloca.h>
+#include <wasi/api.h>
+#include <sysexits.h>
+
+int __main_argc_argv(int argc, char *argv[]);
+
+// If main() uses argc/argv, then no __original_main is emitted, and then
```

Comment:
I thought `__original_main` is no longer emitted by llvm? Or does emitted here mean something else?